controllers/controllers.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # from odoo import http
```

Comment:
Please don't add commented code. Remove this file and all the references to it. When you need this you'll add it.